workspaces/liferay-sample-workspace/client-extensions/liferay-sample-theme-spritemap-2/buildSpritemap.js

```diff
@@ -0,0 +1,72 @@
+const {globSync} = require('glob');
```

Comment:
I'm very interested in finding out what this new path leads us to. It's the first time we move everything to the customer's side (including the build)...

Reply:
yes, this is interesting...

In the case of themecss, build is being handled via workspace plugin. We probably don't want that, I'm fine with this solution. Nevertheless, in the case of spritemap formation, we may consider some way to reuse the existing [logic](https://github.com/liferay/liferay-frontend-projects/blob/95d0e6c9b4ef4586cd17023b3254f79f16134842/projects/js-toolkit/packages/portal-base/src/build/themeSpritemap.ts) we have in the toolkit

Reply:
@dsanz yeah I mostly copied the code from the toolkit. Since our plan is to not use our own custom tools anymore, I think it makes sense to just show here how using existing tools you can extend our existing clay spritemap.

Reply:
yeah in the end that is possible, we're delegating the entire thing to the sample, in this case we are showcasing how to take a set of icons from clay and extend them. But users may as well want to do this in a totally different way...